tests/python/unittest/test_sparse_ndarray.py

```diff
@@ -113,9 +113,20 @@ def check_sparse_nd_csr_slice(shape):
         assert same(A[start:end].asnumpy(), A2[start:end])
         assert same(A[start:].asnumpy(), A2[start:])
         assert same(A[:end].asnumpy(), A2[:end])
+    
+    def check_slice_nd_csr_fallback(shape):
```

Comment:
Could you also add
`from test_sparse_ndarray import test_sparse_nd_slice ` to `tests/python/gpu/test_operator_gpu.py` to make sure it fallbacks on gpu correctly?

Reply:
Added.